src/components/results/results-spine.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import { gmi } from "../../core/gmi/gmi.js";
+
+export class ResultsSpine {
+    constructor(scene, config) {
+        this.config = config;
+        let animation;
+        animation = scene.add.spine(config.offsetX, config.offsetY, config.key, config.animationName, config.loop);
+        scene.add.existing(animation);
+        config.props && Object.assign(animation, config.props);
```

Comment:
You can actually leave out the check - Object.assign(ob, undefined) will just leave as is without failing